File: app/src/main/java/com/reteno/sample/SecondActivity.java
package com.reteno.sample;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import com.reteno.core.RetenoImpl;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        RetenoImpl.Companion.getInstance().pausePushInAppMessages(true);
        new Handler().postDelayed(() -> {
            startActivity(new Intent(this, MainActivity.class));
            finish();
        }, 2000L);
    }

    @Override
    protected void onStop() {
        RetenoImpl.Companion.getInstance().pausePushInAppMessages(false);
        super.onStop();
    }
}

File: app/src/main/java/com/reteno/sample/SecondActivity.kt
package com.reteno.sample

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity
import com.reteno.core.RetenoInternalImpl

class SecondActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
        RetenoInternalImpl.instance.pausePushInAppMessages(true)
        Handler().postDelayed({
            startActivity(Intent(this, MainActivity::class.java))
            finish()
        }, 2000L)
    }

    override fun onStop() {
        RetenoInternalImpl.instance.pausePushInAppMessages(false)
        super.onStop()
    }
}